content/en/docs/reference/glossary/workload.md

```diff
@@ -1,24 +1,23 @@
 ---
-title: Workloads
+title: Workload
 id: workloads
```

Comment:
Note that I kept this as-is to avoid breaking any existing glossary links in the docs.